build.rs

```diff
@@ -136,6 +150,8 @@ fn write_testsuite_tests(
     strategy: &str,
 ) -> anyhow::Result<()> {
     let path = path.as_ref();
+    println!("cargo:rerun-if-changed={}", path.display());
```

Comment:
I guess we don't need them for the files themselves if we load them at runtime, but its nice to have them on the directory so that if you add a new file, you don't have to touch build.rs

Reply:
I personally find mtimes of directories sketchy enough that they're not worth using. It's pretty platform specific what the mtime measures and I think on some platforms it's as bad as printing to rerun on all the contents of the directory. I'd personally prefer to just document the deficiency here and say "touch `build.rs` if you add a new file". We'll always catch it on CI anyway.